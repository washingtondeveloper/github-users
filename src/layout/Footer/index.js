import React from "react";

import "./styles.css";

export default () => (
  <footer className="gs-footer">
    <p>GitUser - {new Date().getFullYear()}</p>
  </footer>
);
