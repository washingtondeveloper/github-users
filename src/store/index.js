import { createStore, applyMiddleware } from "redux";
import reduxThunk from "redux-thunk";

import rootReducer from "./ducks";

export default createStore(rootReducer, applyMiddleware(reduxThunk));
