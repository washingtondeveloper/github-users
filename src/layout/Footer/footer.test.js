import React from "react";

import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "@/store";

import Footer from "./";

describe("Component Footer", () => {
  test("Deve renderizar sem erros", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Footer />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
