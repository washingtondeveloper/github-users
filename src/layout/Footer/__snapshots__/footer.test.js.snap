// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Footer Deve renderizar sem erros 1`] = `
<footer
  className="gs-footer"
>
  <p>
    GitUser - 
    2019
  </p>
</footer>
`;
